### DISCUSSION TITLE
Change attribute box border colour and thickness

### DISCUSSION THREAD

**Discussion by stevef43** [freeplane/freeplane#1437] [Help]
Hi
Is there a way to change the colour, thickness, padding of the box that surrounds **attribute** keys and values?  the colour and thickness seems to be tied to the attribute's node, and the ability to format it when editing styles is greyed out.
Thanks

**Comment by EdoFro** [comment]
As much as I know, colour is the same as the node's border color.
But thickness and padding, I think they are not customizable.

**Reply by stevef43** [reply]
Thanks EdoFro. I appreciate you sharing your knowledge. I guess there is a limit to how far customization is taken for any program.
